src/app/services/auth.service.ts

```diff
@@ -80,4 +80,13 @@ export class AuthService {
       this.router.navigate(['/login']);
     })
   }
+
+  getAuth() { 
+    return this.auth; 
+  } 
+
+  resetPassword(email: string) {
+    return this.auth.sendPasswordResetEmail(email, 
+      { url: 'http://localhost:4200/reset-password' })
```

Comment:
This wouldn't work in production since there's no localhost:4200, can you find another way to route properly? Thanks!

Reply:
@Ammar-T what do you mean by no localhost:4200 though?